assets/styles/style.css

```diff
@@ -0,0 +1,643 @@
+@import url("../styles/fonts/Open_Sans/OpenSans-VariableFont_wdth\,wght.ttf");
+@import url("../styles/fonts/poppins/Poppins-ExtraBold.ttf");
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+}
+
+.sec1 {
+  background-image: url("../image/new\ background.jpg");
+  background-size: cover;
+  background-attachment: fixed;
+}
+
+.logo img {
+  width: 1rem;
+  height: 1rem;
+}
+
+body {
+  font-family: poppins, serif;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+  padding-top: 7rem;
+}
+
+.logo {
+  display: flex;
+  width: 100%;
+  height: 100%;
+  gap: 0.5rem;
+  padding-left: 1rem;
+}
+
+.logo-title {
+  font-size: 1rem;
+  justify-content: center;
+  color: white;
+}
+
+.logo-text {
+  font-size: 1rem;
+  color: white;
+  padding-left: 1rem;
+  padding-bottom: 0.9rem;
+}
+
+.a {
+  display: flex;
+  justify-content: space-between;
```

Comment:
use proper CSS class naming. Or Use a child selector to access children and siblings of a class